autoload/vader/window.vim

```diff
@@ -38,13 +38,16 @@ function! s:switch_to_workbench()
   execute 'b' s:workbench_bfr
 endfunction
 
-function! vader#window#open()
+function! vader#window#close()
   silent! bd \[Vader\]
-  silent! bd \[Vader-workbench\]
+  silent! bw! \[Vader-workbench\]
```

Comment:
My fault. Need of given was related with another pull request to add tap support (I have fixed it already)

https://github.com/albfan/vader.vim/commit/dcceb1ee001e167b9efaf97caa0213f0c23bdb53#diff-ccb3e5fd74f22b318525c8aeade321e8R81

Anyway, need to use buffer wipeout continues. I have split isolation of close function and use of wipeout. Here is a complete scenario:

(I use Vundle for vim plugins)

This fails:

    $ cd ~/.vim/bundle/vader.vim
    $ git checkout close^
    $ cd
    $ git clone https://github.com/albfan/ag.vim
    $ cd ag.vim
    $ cd t
    $ vim 01-aggroup.vader +Vader +VaderClose

This works:

    $ cd ~/.vim/bundle/vader.vim
    $ git checkout close
    $ cd ~/ag.vim/t
    $ vim 01-aggroup.vader +Vader +VaderClose